src/%IPM/Storage/Module.cls

```diff
@@ -1373,66 +1399,81 @@ Storage Default
 <Value>%%CLASSNAME</Value>
 </Value>
 <Value name="2">
-<Value>GlobalScope</Value>
```

Comment:
@Beiming-Zhang Copying classes seems like a waste of time as well though. Instead my suggestion would be to backup the database and restore if it the command fails rather than creating a separate namespace. If you want to copy classes, may as well copy everything like verify and just install the module in the new namespace and do the deploy there (this would probably be the cleanest option actually instead of backing up the database)